spec/ttml1.xml

```diff
@@ -3320,6 +3320,11 @@ above if possible.</p>
 <p>The semantics of the above four cited XSL-FO properties are defined by
 by <bibref ref="xsl11"/>, &sect; 7.17.3, 7.16.7, 7.16.12, and 7.16.8, respectively.</p>
 </note>
+<note role="clarification">
+<p>No presentation semantics are specified for the TAB (U+0009) character. Furthermore, the TAB (U+0009) character 
+is not collapsed when <code>xml:space="default"</code>, and generates a glyph area. As a result, the use of the TAB (U+0009) character 
```

Comment:
If no presentation semantics are defined then surely it is acceptable for a processor to decline to generate a glyph area for a TAB character - in that case change "and generates a glyph area" to "and could generate a glyph area".

Reply:
@nigelmegitt I was using the text that @skynavga had used at https://github.com/w3c/ttml1/issues/235. What about simply removing _ generates a glyph area_?

Reply:
Oh, no, I think it's a worthwhile warning that a glyph area might be generated, but that the details of that glyph area are not defined. I'd leave it in, but modify the statement so it is not potentially self-contradictory.